next.config.mjs

```diff
@@ -23,7 +23,7 @@ export default withBundleAnalyzer(
     env: {
       PAYLOAD_CORE_DEV: 'true',
       ROOT_DIR: path.resolve(dirname),
-      PAYLOAD_DISABLE_DEPENDENCY_CHECKER: 'true',
```

Comment:
Why the rename? Devs may want to use this and having "CI" in it seems to make less sense to me.

Reply:
Not a rename! I still kept the functionality of `PAYLOAD_DISABLE_DEPENDENCY_CHECKER` - but within our own test suite, we want to use `PAYLOAD_CI_DEPENDENCY_CHECKER` (more on that as reply to last comment)